test/converters/junitxml/junitxml.go

```diff
@@ -22,6 +22,7 @@ func (h *Converter) Detect(files []string) bool {
 			h.files = append(h.files, file)
 		}
 	}
+
```

Comment:
Is this blank line required?

Reply:
The return statement is usually not cuddled to other lines if there are more than 2 lines. I am using the "wsl" (white space linter) to check these.

Reply:
👍